magento/magento2
# Admin Config Search

### Description
This feature will provide a possibility to find the config fields in Magento in a fast and easy way (e.g. live search).

### Manual testing scenarios
1. Open Magento admin in your browser.
2. Type the keyword in global search input field.
3. If there is a match in the config elements - results should appear in 'autocomplete results' area.
4. Each result is clickable and should redirect to you to proper section/group of configuration UI.

### Contribution checklist
